pkg/controllers/plugin/util.go

```diff
@@ -193,7 +193,7 @@ func computeReadyCondition(plugin *greenhousev1alpha1.Plugin, pluginStatus green
 		WorkloadReadyStatus.Message = "Following workload resources are not ready: [ "
 		for _, status := range release.PayloadStatus {
 			if !status.Ready {
-				WorkloadReadyStatus.Message += status.Message + ", "
+				WorkloadReadyStatus.Message += ", " + status.Message
 			}
 		}
 		WorkloadReadyStatus.Message += " ]"
```

Comment:
NIT: I think we still have a leading `", "` from the first iteration 
```suggestion
                 WorkloadReadyStatus.Message = strings.TrimPrefix(WorkloadReadyStatus.Message, ", ")
		WorkloadReadyStatus.Message += " ]"
```